doc/deployment/aws_cloudfront.md

```diff
@@ -0,0 +1,78 @@
+# AWS Cloudfront
+
+For a production installation we recommend using cloudfront.
+
+To deploy a production ready AWS cluster:
+
+1) Follow the instructions for a normal AWS deployment
+
+You'll need to use the 'one shot' script. To do that you'll need to clone this repo, cd to it, and run the script:
```

Comment:
Should we just link to the "one shot" instructions in the normal AWS doc here?  Otherwise, we will have to maintain the same instructions in two different places.